src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketsTelemetry.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace System.Net.Sockets
+{
+    [EventSource(Name = "System.Net.Sockets")]
+    internal sealed class SocketsTelemetry : EventSource
```

Comment:
We now have two EventSources in this library.  Is that really what we want?  I'm not clear on why the decision was made to add a new one.  I really don't want to be in a position of having to carry both forward indefinitely.
cc: @noahfalk, @brianrob, @davidsh

Reply:
Generally, you have one `EventSource` per area of functionality.  There's no hard and fast rule, but unless there's a reason to have multiple `EventSources` I'd recommend just having one.  It also makes it easier to consume the events.

Reply:
@alnikola, this doesn't need to block your PR, but I do want us to resolve this for .NET 5. Can you please make sure there's an issue for it?

Reply:
[Issue already exists](https://github.com/dotnet/runtime/issues/37655), but it's outside .NET 5.0 scope.

Reply:
That is related but different.

Reply:
(and if we really want to use that same issue, the description and the milestone should be updated)

Reply:
I created a dedicated issue #38754 for this, but it needs to be triaged to see if it can fit in .NET 5.0 scope.

Reply:
Thanks

Reply:
I added some thoughts to #38754 and may have more once I get a chance to look into this more